crates/terminal/src/terminal.rs

```diff
@@ -1760,11 +1762,24 @@ impl Terminal {
         };
 
         let (finished_successfully, task_line, command_line) = task_summary(task, error_code);
+        // Decide whether to display the task and/or command lines based on user preferences.
+        let mut lines_to_show = Vec::new();
+        if task.hide_task {
+            lines_to_show.push(task_line.as_str());
+        }
+        if task.hide_command {
+            lines_to_show.push(command_line.as_str());
+        }
+
         // SAFETY: the invocation happens on non `TaskStatus::Running` tasks, once,
         // after either `AlacTermEvent::Exit` or `AlacTermEvent::ChildExit` events that are spawned
         // when Zed task finishes and no more output is made.
         // After the task summary is output once, no more text is appended to the terminal.
-        unsafe { append_text_to_term(&mut self.term.lock(), &[&task_line, &command_line]) };
+        // Only append text if there are lines to show
```

Comment:
Passing by and noticing that this comment is useless (as `!..is_empty()` code shows it much better) + the SAFETY comment is not in the right position now (needs to be moved inside the `if` block).

`Decide whether to display the task and/or command lines based on user preferences.` is also useless and not needed.

Reply:
Thanks for the feedback! Appreciate it